### DISCUSSION TITLE
Recognize trace_id and span_id for exemplar record

### DISCUSSION THREAD

**Discussion by jianwu** [grafana/grafana#65141] [Dashboards, Panels & Visualizations]
Current Grafana dashboard only recognizes traceID as trace id in exemplar record. According to [OpenMetrics spec](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md), while it didn't specify the naming convention, it uses `trace_id` in the examples.  And most of common libraries such as `OpenTelemetry`, it also emits `trace_id`. Could we update Grafana to support recognized both `trace_id` and `traceID`.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82755. Feel free to continue the discussion around this there. Thank you!
